kano/utils/hardware.py

```diff
@@ -177,6 +186,8 @@ def get_rpi_model(revision=None):
             model_name = RPI_2_B_KEY
         elif int(revision, 16) & 0x00FFFFFF == 0x00900092:
             model_name = RPI_ZERO_KEY
+        elif revision.strip() == 'a02082':
```

Comment:
You may want to be careful here because of the Overclocking flag. Try to use the same approach as for the other models.